fieldservice/models/fsm_location.py

```diff
@@ -50,9 +50,7 @@ def _tz_get(self):
     calendar_id = fields.Many2one('resource.calendar',
                                   string='Office Hours')
 
-    timezone = fields.Selection(_tz_get, string='Timezone')
-
-    fsm_parent_id = fields.Many2one('fsm.location', string='Parent')
+    parent_id = fields.Many2one('fsm.location', string='Parent')
```

Comment:
@osi-scampbell I may be wrong but `res.partner` already has a `parent_id` field, so there could be a confusion between the `parent_id` from the `res.partner` and this one.